addons/test_mass_mailing/tests/test_mailing_statistics.py

```diff
@@ -26,6 +26,33 @@ def setUpClass(cls):
             signature='--\nMarie'
         )
 
+    @users('user_marketing')
+    @mute_logger('odoo.addons.mass_mailing.models.mailing', 'odoo.addons.mail.models.mail_mail', 'odoo.addons.mail.models.mail_thread')
+    def test_mailing_clicks_ratio(self):
```

Comment:
Just to avoid another setup and data preparation, can't it be included direclty into 'test_mailing_statistics' ?